src/ToDo.UI/Views/Dialogs/ExpirationDateFlyout.xaml

```diff
@@ -13,8 +13,77 @@
 	  Background="{ThemeResource BackgroundBrush}">
 
 	<Page.Resources>
-		<x:String x:Key="Icon_Calendar_Today">F1 M 18 2 L 17 2 L 17 0 L 15 0 L 15 2 L 5 2 L 5 0 L 3 0 L 3 2 L 2 2 C 0.8999999761581421 2 0 2.899999976158142 0 4 L 0 20 C 0 21.100000023841858 0.8999999761581421 22 2 22 L 18 22 C 19.100000023841858 22 20 21.100000023841858 20 20 L 20 4 C 20 2.899999976158142 19.100000023841858 2 18 2 Z M 18 20 L 2 20 L 2 7 L 18 7 L 18 20 Z</x:String>
-		<x:String x:Key="Icon_Chevron_Right">F1 M 1.4099998474121094 0 L 0 1.4099998474121094 L 4.579999923706055 6 L 0 10.59000015258789 L 1.4099998474121094 12 L 7.409999847412109 6 L 1.4099998474121094 0 Z</x:String>
+		<ResourceDictionary>
+			<x:String x:Key="Icon_Add">F1 M 14 8 L 8 8 L 8 14 L 6 14 L 6 8 L 0 8 L 0 6 L 6 6 L 6 0 L 8 0 L 8 6 L 14 6 L 14 8 Z</x:String>
+			<x:String x:Key="Icon_Calendar_Today">F1 M 18 2 L 17 2 L 17 0 L 15 0 L 15 2 L 5 2 L 5 0 L 3 0 L 3 2 L 2 2 C 0.8999999761581421 2 0 2.899999976158142 0 4 L 0 20 C 0 21.100000023841858 0.8999999761581421 22 2 22 L 18 22 C 19.100000023841858 22 20 21.100000023841858 20 20 L 20 4 C 20 2.899999976158142 19.100000023841858 2 18 2 Z M 18 20 L 2 20 L 2 7 L 18 7 L 18 20 Z</x:String>
+			<x:String x:Key="Icon_Chevron_Right">F1 M 1.4099998474121094 0 L 0 1.4099998474121094 L 4.579999923706055 6 L 0 10.59000015258789 L 1.4099998474121094 12 L 7.409999847412109 6 L 1.4099998474121094 0 Z</x:String>
+
+			<Style x:Key="MyCustomDatePickerStyle"
```

Comment:
any particular reason why a custom style was necessary?
and if it is really needed, try to name it accordingly to its function

Reply:
cc @kazo0

Reply:
Yes, the DatePicker in the flyout does not look like the default Material style. You can see it in the ToDo Figma:
![image](https://user-images.githubusercontent.com/4793020/167743757-1d619b08-812f-4341-9249-e1099224d0f7.png)

Reply:
I think we should just be using a simple button with icon, text "Pick a Date" and another icon `>` like the previous two.
And, use a not_mobile:DatePickerFlyout or mobile:NativeDatePickerFlyout for this button's Flyout, and bind a starting value.
As for returning a value, you can hook into DatePickerFlyout.DatePicked to handle that. Best would be to create a behavior to wrap the event into a command. OR... simply two-way bind the date, and detect the changes in values.